js/test/calendar/testpersiandateastro.js

```diff
@@ -635,7 +635,7 @@ module.exports.testpersiandateastro = {
 
         test.ok(pd !== null);
 
-        test.equal(pd.getJulianDay(), 2458199.238194444);
+        test.equal(pd.getJulianDay(), 2458199.1965277777);
```

Comment:
This zoneinfo data changing affects these test cases too?

Reply:
The given persian date is 3/21/2018 in Gregorian. The dates of start and end of DST in the zone info file are in gregorian. The dates of the start and end of DST were calculated wrong before for Persian and were corrected from 3/22 to 3/21. That means that previously, the given date was before the start of DST and with this change, it is after the start of DST. That is the difference in Julian day calculation -- 1 hour. (2458199.238194444 - 2458199.1965277777) * 24 = 1